modules/apps/collaboration/wiki/wiki-web/src/main/resources/META-INF/resources/wiki/page_activity_attachment_action.jsp

```diff
@@ -48,7 +48,7 @@ WikiPage wikiPage = (WikiPage)request.getAttribute(WikiWebKeys.WIKI_PAGE);
 			<portlet:param name="redirect" value="<%= currentURL %>" />
 			<portlet:param name="nodeId" value="<%= String.valueOf(wikiPage.getNodeId()) %>" />
 			<portlet:param name="title" value="<%= wikiPage.getTitle() %>" />
-			<portlet:param name="fileName" value="<%= fileEntry.getTitle() %>" />
+			<portlet:param name="fileName" value="<%= HtmlUtil.unescape(fileEntry.getTitle()) %>" />
```

Comment:
Hey @brianikim I don't think we need to unescape this because fileEntry doesn't seem to be an escaped entity.